app/controllers/facility_journals_controller.rb

```diff
@@ -7,12 +7,13 @@ class FacilityJournalsController < ApplicationController
   before_action :authenticate_user!
   before_action :check_acting_as
   before_action :check_billing_access
-  before_action :init_journals, except: :create_with_search
-  helper_method :has_pending_journals?
+  before_action :init_journals, except: :create
 
   include TransactionSearch
 
-  layout "two_column"
+  layout lambda {
```

Comment:
I really wish there was a nicer way to write this 😞

Reply:
I'm all ears if you have a suggestion--I borrowed that from a few other controllers where we have a similar pattern.